ConfigSpace/util.pyx

```diff
@@ -436,3 +436,185 @@ def fix_types(configuration: dict,
             else:
                 raise TypeError("Unknown hyperparameter type %s" % type(param))
     return configuration
+
+cimport cython

```

Comment:
Could you please put imports at the top of the file?

Reply:
Done